timmys/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Apple-app</h1>
+    <p>
+      Lorem, ipsum dolor sit amet consectetur adipisicing elit. Enim cupiditate
+      alias at mollitia quas perferendis quo autem ab ullam, nobis minus
+      doloribus magnam optio laboriosam voluptates nisi sed vero consectetur.
+    </p>
+    <script src="script.js"></script>
+  </body>
+</html>
```

Comment:
Great progress, @ferdinandchisom 

You have come a long way with the require task. However, there are some changes required to complete the task.

- Go to the task details and copy
   - html
   - css
   - js
 - Update them and resubmit.